aws/resource_aws_sagemaker_notebook_instance_test.go

```diff
@@ -371,3 +413,57 @@ data "aws_iam_policy_document" "assume_role" {
 }
 `, notebookName, notebookName)
 }
+
+func testAccAWSSagemakerNotebookInstanceConfigDirectInternetAccess(notebookName string) string {
+	return fmt.Sprintf(`
+resource "aws_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
+}
+
+resource "aws_subnet" "main" {
+  vpc_id     = "${aws_vpc.main.id}"
+  cidr_block = "10.0.1.0/24"
+
+  tags = {
+    Name = "Main"
```

Comment:
Nit: To prevent confusion for anyone running the acceptance testing in their own accounts it might be good to make this Name tags more descriptive for the test configurations in case the testing leaves dangling resources, e.g.

```suggestion
    Name = "tf-acc-test-sagemaker-notebook-instance-direct-internet-access"
```